keps/sig-node/2008-forensic-container-checkpointing/README.md

```diff
@@ -354,6 +359,14 @@ We expect no non-infra related flakes in the last month as a GA graduation crite
   features enabled with CRI-O, no results have been collected and tests have
   been skipped.
 
+- Since the release of containerd 2.0 and CRI-O 1.25 and since the graduation of
```

Comment:
@SergeyKanzhelev @mikebrow @samuelkarp do you think containerd 2.0 is new enough to consider the checkpoint CRI API GA?